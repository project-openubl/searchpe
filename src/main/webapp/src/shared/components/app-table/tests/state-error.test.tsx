import React from "react";
import { shallow } from "enzyme";

import { StateError } from "../state-error";

describe("StateError", () => {
  it("Renders without crashing", () => {
    const wrapper = shallow(<StateError />);
    expect(wrapper).toMatchSnapshot();
  });
});
