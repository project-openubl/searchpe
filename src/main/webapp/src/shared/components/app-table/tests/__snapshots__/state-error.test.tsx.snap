// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StateError Renders without crashing 1`] = `
<EmptyState
  variant="small"
>
  <EmptyStateIcon
    color="#a30000"
    icon={[Function]}
  />
  <Title
    headingLevel="h2"
    size="lg"
  >
    Unable to connect
  </Title>
  <EmptyStateBody>
    There was an error retrieving data. Check your connection and try again.
  </EmptyStateBody>
</EmptyState>
`;
